playgrounds/react-vite/src/App.tsx

```diff
@@ -1,59 +1,58 @@
-import { Box, Button } from "@0xsequence/design-system";
-import { useOpenConnectModal } from "@0xsequence/kit";
-import {
-  CollectibleCard,
-  useCreateListingModal,
-  useMakeOfferModal,
-} from "@0xsequence/marketplace-sdk/react";
-import { useAccount } from "wagmi";
-import { sdkConfig } from "./config";
-import Providers from "./provider";
-
-const consts = {
-  collectionAddress: "0xf2ea13ce762226468deac9d69c8e77d291821676",
-  chainId: "80002",
-  collectibleId: "1",
-} as const;
+import { Box, Divider, Tabs, Text } from "@0xsequence/design-system";
+import { type Tab, useMarketplace } from "./lib/MarketplaceContext";
+import Providers from "./lib/provider";
+import { Collections } from "./tabs/Collections";
+import { Collectible } from "./tabs/Collectable";
+import { Collectibles } from "./tabs/Collectables";
+import { Settings } from "./lib/Settings";
 
 function App() {
   return (
-    <Providers sdkConfig={sdkConfig}>
-      <InnerApp />
+    <Providers>
+      <div style={{ width: "100vw", paddingBlockStart: "70px" }}>
```

Comment:
What about having some space for the bottom as well?

Reply:
Yeah, that better added